Data Model/parishName.js

```diff
@@ -0,0 +1,70 @@
+export const data =
+    [
+        {
+            id: null,
+            type: 'property',
+            datatype: 'wikibase-item',
```

Comment:
is any difference between datatype: 'wikibase-item' and 'item' in line 57?

Reply:
I think that there is some technical reason. Based on the [documentation](https://www.wikidata.org/wiki/Help:Data_type)

 ![CleanShot 2024-01-03 at 09 19 06](https://github.com/ontohgis/wikikul/assets/92399615/f85afb1c-fb0f-4bf6-9cc7-ad0a965bc434)